src/include/daos_task.h

```diff
@@ -49,7 +49,7 @@ typedef enum {
 	/** Management APIs */
 	DAOS_OPC_SVC_RIP = 0,
 	DAOS_OPC_POOL_EXTEND,
-	DAOS_OPC_POOL_EVICT,
+	DAOS_OPC_RESERVED3,
```

Comment:
I think rather than adding dummy entries, we can just do something like...

DAOS_OPC_SET_PARAMS = /* expected value */, and skip the old entry